CICD/github/launch_karate_tests.sh

```
@@ -20,12 +20,12 @@ cd bin
 ./startOpfab.sh prod
 #Logs for debug
 docker logs greenmail
-cd ../src/test/api/karate
+cd ../test/api/karate
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)